<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,10,0,10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="B_scan">
        <property name="text">
         <string>scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="B_connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="B_Config">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="B_Start">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_port_1">
        <property name="text">
         <string>Node 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_portSelect_1"/>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_port_2">
        <property name="text">
         <string>Node 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_portSelect_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_portSelect_3"/>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_portSelect_4"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_port_3">
        <property name="text">
         <string>Node 3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_port_4">
        <property name="text">
         <string>Node 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_min_cali">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-1 calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_min_action">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-1 action</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="m_label_2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_lable_1">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_cali">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Action</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
